packages/react-components/src/components/Select/Select.unit.spec.jsx

```diff
@@ -200,7 +213,8 @@ describe('<Select>', () => {
         onValueChange={() => {}}
       />,
     );
-    expect(tree.find('label').text()).to.equal('my label(*Required)');
+    expect(tree.find('label').text()).to.include('my label(*Required)');
```

Comment:
Now that we're doing the `i18next.init` we can probably leave the `equal`:

```suggestion
    expect(tree.find('label').text()).to.equal('my label(*Required)');
```